### PR TITLE
GROW-640 Partner details

### PR DESCRIPTION
**Pull request by mlibazisi**

### CODE REVIEW

File: src/components/BorrowerProfile/FieldPartnerDetails.vue [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                    <button class="tw-underline">
+                        Field Partner risk rating
+                    </button>
+                </dt>
+                <dd>
+                    <div class="tw-flex">
+                        <kv-material-icon
+                            v-for="i in 5"
+                            :key="i"
+                            class="tw-text-black hover:tw-text-action-700"
```
Comment (ryan-ludwig) [inline src/components/BorrowerProfile/FieldPartnerDetails.vue:46]: Since these aren't interactive, I don't think we want the `hover:tw-...` class

**Review by ryan-ludwig — APPROVED** [review]
👍

**Review by emuvente — APPROVED** [review]
:+1:

**Review by BoulderBrains — COMMENTED** [review]
👀

**Pull request merged**